docs/adding-results.md

```diff
@@ -0,0 +1,76 @@
+  `config.yaml`.
+
+- **`docker-run.sh`**: Runs the container, taking the dataset path as an argument. Typically, only
+  the `container_name` variable needs alignment with `container_id` in `config.yaml`.
+
+- **`Dockerfile`**: Used for building the container, ensuring installation of the required tool and
+  dependencies.
+
+- **`launch.sh`**: Initializes and starts the tool (e.g., if it functions as a server or
+  service).
+
+- **`reset.sh`**: Prepares a clean environment by removing previous data (e.g., dropping
+  tables); runs after `launch.sh` in `ingest` mode.
+
+- **`measure-decompressed-size.sh`**: Measures the raw dataset size before ingestion.
+  Typically unchanged, it takes `datasets_path` from `config.yaml` and uses `du -bc` for size
+  calculation in bytes.
+
+- **`ingest.sh`**: Handles data ingestion, with clp-bench measuring the total latency of this
+  script. Avoid adding extra operations.
+
+- **`measure-compressed-size.sh`**: Measures the compressed data size post-ingestion, usually
+  via tool-specific methods.
+
+- **`search.sh`**: Executes queries specified in `config.yaml`. clp-bench supports two
+  benchmarking modes:
+
+  - **Hot-run mode**: Runs queries for `hot_run_warm_up_times` to warm up the cache, then measures
+    latency.
+  - **Cold-run mode**: Clears the cache with `clear-cache.sh` before measuring latency.
+
+- **`clear-cache.sh`**: Clears the tool's cache, essential for cold runs.
+
+- **`methodology.md`**: Describes specific benchmarking set up details, including tuning and dataset
+  preprocessing.
+
+- **`results.json`**: Contains benchmarking results, which are loaded and displayed in the UI:
+
+  - **`target`**: The ID used by the frontend, should be lowercase. IDs of the same type must be
+    unique.
+  - **`targetDisplayedName`**: The name to display in the column on the webpage.
+  - **`displayedOrder`**: Defines the display order of results; a smaller value places the column
+    further to the right.
+  - **`isEnable`**: Indicates if the results should be displayed (default is `true`). If set to
+    `false`, results won't appear on the webpage.
+  - **`type`**: Specifies data type (1 for Unstructured, 2 for Dynamically-structured).
+  - **`ingestTime`**: Total end-to-end time taken to ingest all dataset data.
+  - **`compressedSize`**: The size of compressed archives.
+  - **`avgIngestMem`**: The average memory used during ingestion.
+  - **`metrics`**: An array of query benchmarking results for each metric:
+
+    - **`metric`**: Specifies the type (1 for Hot run, 2 for Cold run).
+    - **`avgQueryMem`**: The average memory usage during query benchmarking.
+    - **`queryTimes`**: An array of end-to-end query latencies, ordered to match the sequence of
+      queries.
+- **binary (optional)**: If the benchmarked tool is run via binary, then you should put a 
+  `replace-with-your-tool-binary` file in the assets. 
```

Comment:
I feel like this file is kind of confusing. If the user sets up launch correctly, I feel like this would be obvious to them, right?

Reply:
Right, but here I think it might be better if it can keep consistent with `assets/unstructured/glt` or `assets/dynamically-structured/clp-s`, where the `launch.sh` scripts will use the binaries under the same directory. So the point of this file is that I don't want user to mistakenly upload the entire binary to the repo so I add this file.

Yea, if you think this is redudant I can get rid of it.

Reply:
I think since we control the repo, we should be able to stop folks from uploading the binary to the repo. The methodology for clp-s and glt, both indicate that the binary should be downloaded or compiled; so similarly, I think a new user would write a similar methodology for their own tool.